templates/admin_event/edit.html.twig

```diff
@@ -0,0 +1,13 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Edit Event{% endblock %}
```

Comment:
plutot ecrire en francais pour le modo

edit->modifier
back to list-> retour a la liste etc...